adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
+from .constants import LETTER_POOL, SCORE_CHART, ALPHABET
 
 def draw_letters():
-    pass
+
+    alphabet_pool = dict(LETTER_POOL)
+    results = []
+
+    while len(results) < 10:
+
+        random_num = randint(0, 25)
+
+        letter = ALPHABET[random_num]
+
+        if alphabet_pool[letter] > 0:
+            results.append(letter)
+            alphabet_pool[letter] -= 1
+
+    return results
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    
+    cap_word = word.upper()
```

Comment:
Prefer to call `upper` directly on `word` in the for loop so we don't need to write an extra line of code. More lines of code make a codebase more difficult to maintain and makes for more places a bug can be accidentally introduced.

We'd remove line 24 here and update the loop on line 28, like
```py
    for letter in word.upper():